src/ekore/operator_matrix_elements/polarized/space_like/as2.py

```diff
@@ -9,8 +9,23 @@
 
 from ....harmonics.constants import zeta2, zeta3
 from ....harmonics.polygamma import cern_polygamma
+from....anomalous_dimensions.unpolarized.space_like.as1 import gamma_ns as gamma0_qq 
+from....anomalous_dimensions.polarized.space_like.as1 import gamma_gq as gamma0_gq
+from....anomalous_dimensions.polarized.space_like.as1 import gamma_qg as  gamma0_qg
+from....anomalous_dimensions.polarized.space_like.as1 import gamma_gg as  gamma0_gg
 
+Beta_0Q = -4 / 3 * constants.TR
 
+@nb.njit(cache=True)
+def Beta_0(nf):
```

Comment:
This is [`beta_qcd_as2`](https://github.com/NNPDF/eko/blob/45682d7775210f262529d0f639fd1d21a48b5457/src/eko/beta.py#L14), no?